src/components/Activities/Activity1/Game1/components/CipherGameRound.tsx

```diff
@@ -56,19 +56,31 @@ function CipherGameRound(props : CipherGameRoundProps): JSX.Element {
   const [slideIdx, setSlideIdx] = useState(0);
   const [happiness, setHappiness] = useState(0);
   const [clickDisabled, setClickDisabled] = useState(false);
+  const [hoverIncorrect, setHoverIncorrect] = useState(false);
+  const [roundNum, setRoundNum] = useState(0);
+
+  useEffect(()=>{
+    if (roundNum >   THRESHOLD_TO_HELP_PER_GAME ) {
+      hoverIncorrect ? handleAlienState(ALIEN_STATE.SAD) : handleAlienState(ALIEN_STATE.BASE);
+    }
+  }, [hoverIncorrect]);
```

Comment:
nit: i generally try to eliminate nesting where possible, you don't have to change! but just wanted to share what I do

```suggestion
  useEffect(()=>{
    if (roundNum <= THRESHOLD_TO_HELP_PER_GAME) return;
    handleAlienState(hoverIncorrect ? ALIEN_STATE.SAD : ALIEN_STATE.BASE);
  }, [hoverIncorrect]);
```

Reply:
just wondering, why eliminate nesting where possible? is it for like cleanness or easier to read code?

Reply:
its a rule of thumb that leads to cleaner code

I think there are also other optimization when it comes to the V8 engine, but its too big brain for me

Reply:
ahh right the loop unrolling stuff from cs33, makes sense ty!